src/Parser/Core/Modules/Abilities.js

```diff
@@ -29,6 +29,7 @@ class Abilities extends Analyzer {
      * Available properties:
      *
      * required spell {object} The spell definition with { id, name and icon }
+     * optional additionalSpells {array} An array of additional spell definitions with { id, name and icon }. Can be used if an ability has multiple cast IDs, or to associate a buff ID with a cast ID. The 'spell' field is still required even if you define this, as the spell field is used for the user facing display of the ability.
```

Comment:
I think instead of adding another property, spell should also allow for an array to be passed to make usage simpler.

Reply:
I know it looks awkward, but I did it this way for two reasons:
1) Makes it possible to specify the 'main' spell when showing user facing elements. See in the image above the suggestion is about Grimoire of Service because that's the one in the 'spell' field.
2) Easier on dev, don't have to check type of spell every time I use it.

Looking at usage of ABILITIES, it's not currently touched from that many places, so I could change it. I guess I'd specify the 'main' spell by whichever is first in the array?

Reply:
I'd rather have spell be of type `object|arrayOf(object)` with a second property indicating which is main, than have spell mean two things and having a second property for extending it if you get what I mean.

Selecting first would make sense I think. I can't think of a situation where the main spell can't be first.

Reply:
Technically the spell field doesnt mean two things, because it counts as one of the IDs to be tracked even if additionalSpells is defined. 

But ok ok, I'll change it.